src/app/static/src/js/components/admin/refreshDocuments.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div>
+        <p>Enter a Dropbox share link to download files from</p>
```

Comment:
Is this too specific? Doesn't need to be a Dropbox link.

Reply:
Yeah I was initially going to say something like "Enter a url to a zipped resource" - but not sure how comprehensible that is to an admin user, also not sure what other types of link it could realistically be, can you think of any others? Perhaps one option is to have a smaller help text saying something like "or any url that points to a zipped folder" - any ideas for phrasing?

Reply:
Yeah, not sure since we're only considering the Dropbox use case for now! Maybe
Label: Download files from: 
Help text: Provide a link to the zipped document folder, e.g. Dropbox share link
?